src/components/package/show/package-show.js

```diff
@@ -126,57 +134,113 @@ class PackageShow extends Component {
 
     const { packageSelected } = this.state;
 
+    const isAddButtonNeeded = !packageSelected || model.isPartiallySelected;
+
     return (
       <Fragment>
-        <Button
-          buttonStyle="dropdownItem fullWidth"
-          onClick={onEdit}
-        >
-          <FormattedMessage id="ui-eholdings.actionMenu.edit" />
-        </Button>
-
-        {onFullView && (
+        <IfPermission perm="ui-eholdings.records.edit">
```

Comment:
Why did you chabnge the order of buttons in the dropdown? was it according to  acceptance criteria?

Reply:
I didn't change it. I think git shows the diffs in a confusing way